climada/hazard/tc_tracks.py

```diff
@@ -625,19 +671,24 @@ def _read_file_emanuel(self, path, hemisphere='S', rmw_corr=False):
         path : str
             absolute path of file to read.
         hemisphere : str, optional
-            'S', 'N' or 'both'. Default: 'S'
+            For global data sets, restrict to northern ('N') or southern ('S') hemisphere.
+            Use 'both' to disable this restriction. Default: 'S'
```

Comment:
I know I am nitpicking, but actually, the string `both` is never used in the code.